src/telliot_feeds/cli/utils.py

```diff
@@ -42,7 +69,7 @@ def build_feed_from_input() -> Optional[DataFeed[Any]]:
     """
     try:
         query_type = input("Enter a valid Query Type: ")
-        feed = DATAFEED_BUILDER_MAPPING[query_type]
+        feed: DataFeed[Any] = DATAFEED_BUILDER_MAPPING[query_type]
     except KeyError:
         click.echo(f"No corresponding datafeed found for Query Type: {query_type}\n")
```

Comment:
Small thing, but you have inconsistent naming/capitalization in this function.
Change:
`Query Type` -> `Query type`
`Query Parameter` -> `Query parameter`
`QueryType` -> `Query type`
`QueryParameter` -> `Query parameter`